packages/ove-app-networks/src/client/common/networks.js

```diff
@@ -13,6 +13,235 @@ $(function () {
     });
 });
 
+runOperation = function (message) {
+    // Helper method to retrieve a property from an element
+    const getFromElement = function (e, x) {
+        let m;
+        let n;
+        if (!x || x.indexOf('.') === -1) {
+            n = x;
+        } else {
+            m = x.substring(x.indexOf('.') + 1);
+            n = x.substring(0, x.indexOf('.'));
+        }
+        const p = e[n] || (e.attributes ? e.attributes[n] : undefined);
+        return m ? getFromElement(p, m) : p;
```

Comment:
I think the logic would be clearer if the ternary was replaced by moving the logic into if statement:

```JavaScript
        let m, n;
        if (!x || x.indexOf('.') === -1) {
            return e[x] || (e.attributes ? e.attributes[x] : undefined);
        } else {
            m = x.substring(x.indexOf('.') + 1);
            n = x.substring(0, x.indexOf('.'));
            const p = e[n] || (e.attributes ? e.attributes[n] : undefined);
            return getFromElement(p, m);
        } 
```